styles.css

```diff
@@ -13,7 +13,30 @@ body {
 }
 
 .logo {
+  position: relative;
   padding: 40px 0;
+  padding-left: 44px;
+  font: bold 24px/31px "pt_serif", arial, helvetica, sans-serif;
+}
+
+.logo__circle {
+  position: absolute;
+  left: 0;
+  top: 50%;
+  transform: translateY(-50%);
+  width: 28px;
+  height: 28px;
+  opacity: .7;
+}
+
+.logo__circle_color_dark-blue {
+  background: #0056EC;
```

Comment:
Codacy found an issue: [Expected "#0056EC" to be "#0056ec" (color-hex-case)](https://app.codacy.com/gh/WebHeroSchool/newprojectA/pullRequest?prid=6326212)